COPYRIGHT.md

```diff
@@ -0,0 +1,4 @@
+# Copyright
+
+- Copyright (c) 2020-2024 Bernd Deichmann
```

Comment:
remove the "Copyright (c)" for all entries

Reply:
done